platform/darwin/scripts/generate-style-code.js

```diff
@@ -77,17 +83,33 @@ global.testHelperMessage = function (property, layerType, isFunction) {
     }
 };
 
-global.propertyDoc = function (property, layerType) {
-    let doc = property.doc.replace(/`(.+?)`/g, function (m, symbol, offset, str) {
+global.propertyDoc = function (propertyName, property, layerType) {
+    // Match references to other property names & values. 
+    // Requires the format 'When `foo` is set to `bar`,'.
+    let doc = property.doc.replace(/When `(.+?)` is set to `(.+?)`,/g, function (m, peerPropertyName, propertyValue, offset, str) {
```

Comment:
Is "When" needed? If so, can it be case-insensitive using the `i` flag? Is the comma needed? I think we should try to avoid coupling too tightly to how things are worded right now, because the style spec repo doesn't have any documentation that would inform a contributor that they need to worry about these details.